source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidNearbyConnectionClient.cs

```diff
@@ -0,0 +1,399 @@
+#if UNITY_ANDROID
+
+namespace GooglePlayGames.Android
+{
+    using System;
+    using System.Linq;
+    using System.Collections.Generic;
+    using System.Threading;
+    using GooglePlayGames.BasicApi;
+    using GooglePlayGames.BasicApi.Nearby;
+    using GooglePlayGames.OurUtils;
+    using UnityEngine;
+    using UnityEngine.Android;
+    using UnityEngine.SocialPlatforms;
```

Comment:
Is UnityEngine.SocialPlatforms used for nearby?

Reply:
Removed